watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/tests/test_multi_sweep.py

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import pytest
+from watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.metab.multi_sweep import *
+
+pytest_parameterize_list = []
+# We are skipping cases 5 & 7 in order to keep the pytests times in check
```

Comment:
```suggestion
```
Misleading comment

Reply:
Oh yeah that is leftover from the full treatment train test